### ISSUE TITLE
RFC: Request for creation of Contribution Guideline

### ISSUE THREAD

**Issue by Jagatmachines** [0xmetaschool/Learning-Projects#18]
# RFC: Request for creation of Contribution Guideline

## Context
While we are trying to contribute to this project, there don't seem to be contribution guidelines. It would be great if we could create one and encourage the community to follow it.

**Comment by VipinVIP** [comment]
Yes. There are so many Pull Requests pending and, most of them can only be understood from reading the file changelog manually. Having a good Contribution guidelines would be super nice.
